code/client/clrcore/Math/Vector4.cs

```diff
@@ -1442,5 +1442,26 @@ public override bool Equals(object value)
 			var strongValue = (Vector4)value;
 			return Equals(ref strongValue);
 		}
+
+		/// <summary>
+		/// Converts the specified <see cref="CitizenFX.Core.Vector4"/> to an Array float[4]
+		/// </summary>
+		/// <param name="vector"></param>
+		/// <returns>
+		/// A new float[4]
+		/// </returns>
+		public static float[] ToArray()
+		{
+			try
+			{
+				return new float[] { X, Y, Z, W };
+			}
+			catch (Exception ex)
+			{
+				Debug.WriteLine($"ToArray exception: {ex.Data}");
+			}
+			return null;
+		}
+

```

Comment:
what's the point of using the try-catch block?
Here would make sense because of the `static` method, but Vector4 is a struct and can't be null.

Reply:
i've removed the ToArray natives as they're actually already in the API.. i didn't see it 😅